### PR TITLE
 [SCB-1193]upgread spring boot versions and add easy to use dependency

### PR DESCRIPTION
**Pull request by liubao68**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install -Pit` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by liubao68** [comment]
examples are here: https://github.com/apache/servicecomb-samples/tree/master/dependency_management

**Comment by wujimin** [comment]
too many logs
maybe we can close all generated swagger log in ProducerSchemaFactory
```
  <logger name="org.apache.servicecomb.core.definition.schema.ProducerSchemaFactory" level="OFF"/>
```

**Comment by wujimin** [comment]
> examples are here: https://github.com/apache/servicecomb-samples/tree/master/dependency_management

good job!!

File: integration-tests/it-consumer/src/test/resources/logback.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (wujimin) [inline integration-tests/it-consumer/src/test/resources/logback.xml]: already have one logback.xml in it-common

File: samples/bmi/pom.xml [hunk 2]
```diff
@@ -69,6 +69,7 @@
       <plugin>
         <groupId>org.apache.rat</groupId>
         <artifactId>apache-rat-plugin</artifactId>
+        <version>0.13</version>
```
Comment (WillemJiang) [inline samples/bmi/pom.xml]: It's not a good practice to specify the plugin version here.

File: parents/spring4/pom.xml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  <artifactId>java-chassis-parent-spring4</artifactId>
+  <packaging>pom</packaging>
+
+  <name>Java Chassis::Parent::Spring4</name>
+  <description>Java Chassis Parent Spring4</description>
+  <url>https://github.com/apache/servicecomb-java-chassis</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```
Comment (WillemJiang) [inline parents/spring4/pom.xml]: Can we put the properties into some common place?

File: samples/bmi/pom.xml (hunk 2 above)
Comment (liubao68) [reply samples/bmi/pom.xml]: Why? Any suggestions?

Comment (WillemJiang) [reply samples/bmi/pom.xml]: you can use the property variable to manage the version in the single place, just like put the <version>${rat.version}</version> here.

**Pull request merged**